nipype/interfaces/dipy/base.py

```diff
@@ -110,7 +110,7 @@ def convert_to_traits_type(dipy_type, is_file=False):
     """Convert DIPY type to Traits type."""
     dipy_type = dipy_type.lower()
     is_mandatory = bool("optional" not in dipy_type)
-    if "variable" in dipy_type and "string" in dipy_type:
+    if "variable" in dipy_type and "str" in dipy_type:
```

Comment:
I tested this on 1.5.0 (which was not failing) and 1.6.0 (which was). Both pass.